src/test/java/com/butbetter/storage/model/ProductInformationTests.java

```diff
@@ -0,0 +1,65 @@
+package com.butbetter.storage.model;
+
+import org.junit.jupiter.api.*;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Date;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class ProductInformationTests {
+
+    @BeforeEach
+    void setUp() {
+
+    }
+
+    @BeforeAll
+    static void beforeAll() {
+
+    }
+
+    @AfterAll
+    static void afterAll() {
+
+    }
+
+    @AfterEach
+    void tearDown() {
```

Comment:
if not needed, it can be removed

Reply:
auto generated

Reply:
if you don't need it, you don't need to generate it